e2e/__tests__/configOverride.test.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {getConfig} from '../runJest';
+
+test('reads config from cjs file', () => {
```

Comment:
I'd like a snapshot of `stderr` so we can see the warning when the `--config` flag is not passed 🙂 (essentially what the unit test asserts, but without any mocks etc)

Reply:
:+1: for adding a snapshot of the error text.

But it should be probably in a different test because `config-override` has only one config, so it won't throw without `--config`.

I will create an unit test with a snapshot for this. I think that is a better tool for the job than an e2e test.

Reply:
Added snapshots for 2-6 config files.